packages/dialog/dialog-wrapper.md

```diff
@@ -0,0 +1,86 @@
+## Description
+
+`sp-dialog` display important information that users need to acknowledge. They appear over the interface and block further interactions.
```

Comment:
I think you may have forgotten to change this to `sp-dialog-wrapper`

Reply:
Good catch!